.github/workflows/deploy.yml

```diff
@@ -55,21 +59,61 @@ jobs:
             - name: 🔨 Build
               run: npm run build
             
-            - name: 🔗 Supabase Link
+            - if: ${{ env.IS_SUPABASE_CONFIGURED }}
+              name: 🔗 Supabase Link
               run: npx supabase link --project-ref $SUPABASE_PROJECT_ID
 
-            - name: 📡 Push supabase migrations
+            - if: ${{ env.IS_SUPABASE_CONFIGURED }}
+              name: 📡 Push supabase migrations
               run: npx supabase db push
 
-            - name: 📡 Push supabase functions secrets
+            - if: ${{ env.IS_SUPABASE_CONFIGURED && env.POSTMARK_WEBHOOK_USER }}
+              name: 📡 Push supabase postmark webhook user secret
+              run: |
+                  npx supabase secrets set POSTMARK_WEBHOOK_USER=${{ env.POSTMARK_WEBHOOK_USER }}
+
+            - if: ${{ env.IS_SUPABASE_CONFIGURED && env.POSTMARK_WEBHOOK_PASSWORD }}
```

Comment:
Why split it into two steps?

Reply:
This is to only push the key we have.
This way if we only have one of the key, we push it and ignore the other.
I can revert if you think that's overkill.

Reply:
yes, I think it's overkill as we need to push all the postmak conf if it's available